sdk/identity/Azure.Identity/src/DefaultAzureCredentialOptions.cs

```diff
@@ -51,6 +51,11 @@ public class DefaultAzureCredentialOptions : TokenCredentialOptions
         /// </remarks>
         public string SharedTokenCacheUsername { get; set; } = GetNonEmptyStringOrNull(EnvironmentVariables.Username);
 
+        /// <summary>
+        /// Specifies the client id of the selected credential
+        /// </summary>
+        public string InteractiveBrowserCredentialClientId { get; set; }
```

Comment:
should we default this to `Constants.DeveloperSignOnClientId` since that is the client id which will be used if the user doesn't set this?

Reply:
We do that here: https://github.com/Azure/azure-sdk-for-net/blob/a6d39f7245605cc8d5c14cb955c007381708c15e/sdk/identity/Azure.Identity/src/DefaultAzureCredentialFactory.cs#L40